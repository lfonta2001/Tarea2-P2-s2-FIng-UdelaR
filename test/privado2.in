crearCliente 100 Juan Martinez 30
imprimirCliente
imprimirEdadCliente
imprimirIdCliente
copiarCliente
imprimirCliente
liberarCliente
crearCliente 20 Aureliano Gomez 29
imprimirNombreYApellidoCliente
liberarCliente
Fin
