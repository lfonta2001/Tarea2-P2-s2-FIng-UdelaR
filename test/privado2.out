Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Cliente creado de forma exitosa.
2> Cliente Juan Martinez
Id: 100
Edad: 30
3> La edad del cliente es: 30
4> El id del cliente es: 100
5> Cliente copiado. Datos de la copia:
Cliente Juan Martinez
Id: 100
Edad: 30
6> Cliente Juan Martinez
Id: 100
Edad: 30
7> Cliente liberado con exito.
8> Cliente creado de forma exitosa.
9> El nombre del cliente es: Aureliano
El apellido del cliente es: Gomez
10> Cliente liberado con exito.
11> Fin.
